src/docs/community/migration.mdx

```diff
@@ -0,0 +1,242 @@
+import { Divider } from '../../app/components';
+
+<!-- KEYWORDS: developer develop environment guide tutorial -->
+
+# Package Migration Notice
+
+## @brightlayer-ui/angular-auth-workflow
+
+-   `<pxb->` preface changed to `<blui->`.
+-   Change Auth Module from `PxbAuthModule` to `BluiAuthModule`.
+
+## @brightlayer-ui/angular-components
+
+-   `<pxb->` preface changed to `<blui->`
+-   For e.g. App Bar component name change from `<pxb-app-bar>` to `<blui-app-bar>`.
+
+## @brightlayer-ui/angular-template-authentication
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   You can use these templates with the Brightlayer UI CLI:
+
+-   ```sh
+      npx -p @brightlayer-ui/cli blui new angular --template=authentication
+    ```
+
+## @brightlayer-ui/angular-template-blank
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   You can use Blank Template with the Brightlayer UI CLI with following command:
+
+-   ```sh
+      npx -p @brightlayer-ui/cli blui new angular --template=blank
+    ```
+
+## @brightlayer-ui/angular-template-routing
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   You can use Routing Template with the Brightlayer UI CLI with following command:
+
+-   ```sh
+      npx -p @brightlayer-ui/cli blui new angular --template=routing
+    ```
+
+## @brightlayer-ui/angular-themes
+
+-   `<pxb->` preface changed to `<blui->`.
```

Comment:
Is this a change that users have to make? Or is this just a change we made in our scss?